lib/segment/src/index/hnsw_index/hnsw.rs

```diff
@@ -569,9 +594,9 @@ impl<TGraphLinks: GraphLinks> VectorIndex for HNSWIndex<TGraphLinks> {
 
         VectorIndexSearchesTelemetry {
             index_name: None,
-            unfiltered_plain: Default::default(),
+            unfiltered_plain: tm.unfiltered_plain.lock().get_statistics(),
```

Comment:
It looks like `VectorIndexSearchesTelemetry::unfiltered_plain` is unused.

`HNSW::search()` may now do a unfiltered full scan search, if the number of deleted points is high.

Since the telemetry field is unused I figured I could use it for reporting that search method.

Reply:
This is alright as discussed in call.